build/cli.js

```diff
@@ -267,6 +286,7 @@ program
         ).toFixed(1)} documents per second.`
       );
       console.log(`Peak heap memory usage: ${humanFileSize(peakHeapBytes)}`);
+      console.log(formatTotalFlaws(totalFlaws));
```

Comment:
Do we want to keep these? I know Webpack can auto strip these out when building for production but, perhaps you just overlooked these.

Reply:
No, it's deliberate. It's not debugging. It's part of the useful output of the CLI tool.